lib/actions/auth.ts

```diff
@@ -109,3 +110,65 @@ export const register = async (
     };
   }
 };
+
+//the following section defines the functions to recieve and verify the positions of users.
+
+
+//This is the return type of the fuction
+export type User={
+  id:number,
+  name:string|null,
+  organisation:string | null,
+  role:string | null,
+  department:string| null
+}
+
+
+//pass the jwtToken of the user that you want to get the position of.
+export async function getPosition(JWTtoken:string):Promise<{status:number,user:User|null}> {
+  try{
+    //getting user id from token
+    const jwtParsed=jwt.decode(JWTtoken) as jwt.JwtPayload;
+    const userId=jwtParsed.id;
+    const userEmail=jwtParsed.email;
+    //find user info from DB using id
+    try{
```

Comment:
Prisma does not throw any error if it is not able to find a record. It throws an error only if it is not able to reach db